rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,444 @@
+
+**[Epics](http://agiledictionary.com/309/epic/)** 
+are a collection of User Stories required to complete an objective.
+
+**[Spikes](http://agiledictionary.com/209/spike/)** are adopted from eXtreme Programming for activities that require research or design before estimation and final implementation can occur.
+- [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+**Milestones** represent a four week timeline/duration in a Release.
+
+**Releases** are composed of three milestones and occur on a quarterly cadence.
+
+## Detailed Design for Roadmaps
+
+The Product Roadmap creates a shared understanding across the DCP community and Users of the DCP for investments and direction for the next year. The Roadmap process MUST occur on an annual basis. 
+
+Themes and Objectives from the Roadmap are the primary source of priorities for planning Releases each quarter.
+
+### Timebox for Roadmaps
+
+All timeboxes are noted as business days. Business days are included regardless of national holidays.
+
+Overlapping timeboxes denote concurrent work.
+
+Timeboxes with a range of days indicate that the work MAY be completed at any point during the timebox, but MUST be completed no later than the final day of the timebox.
+
+**Note**: *Input from the Technical Architecture team is integrated on a quarterly basis during Release planning.*
+
+Day 1 in the Roadmap timeline begins in the third quarter.
+
+* **Day 1 - Day 30: [Gathering and reviewing input](#gathering-and-reviewing-input)**
+  * Prerequisite: Science Roadmap
+  * Prerequisite: Technical Objectives RFC
+  * Day 1-30: DCP User Survey 
+  * Day 1-10: Roadmap Retrospective
+* **Day 1 - Day 45: [Drafting the roadmap](#drafting-the-roadmap)**
+  * Day 1-10: Identify a Roadmap Shepherd
+  * Day 30-35: Create themes
+  * Day 36: Collate themes
+  * Day 37-45: Refine themes
+* **Day 45 - Day 60: Receiving feedback**
+  * Day 45: Publish draft Roadmap
+  * Day 46-55: Community Review period 
+  * Day 56-60: Complete final edits
+* **Day 60 - Day 70: [Iterating with Oversight](#iterating-with-oversight)**
+  * Day 60: Send Roadmap to Oversight
+  * Day 61-70: Review and edit roadmap with Oversight
+  * Day 70: Receive Roadmap approval from Oversight
+  * Day 70: Communicate approved Roadmap 
+
+
+### Gathering and Reviewing Input
+[Gathering and Reviewing Input]:#gathering-and-reviewing-input
+
+#### Science Roadmap
+
+Oversight MUST provide an annual Science Roadmap to Project Leads to be reviewed at the next Oversight meeting following its receipt.
+
+Project Leads MUST clarify outstanding questions about the Science Roadmap with Oversight within two weeks.
+
+#### Technical Objectives RFC
+
+Project Leads MUST review the [Technical Objectives RFC](#technical-architecture-objectives) from the most recent quarter to understand outstanding technical concerns.
+
+#### DCP User Survey and UX Insights
```

Comment:
Hi, @brianraymor and @CaraLMason

As Brian suggested before, I am placing here my view on what I think the UX insights section should be comprised of.
I'm here for any comments. Also - this should be the basis of the UX charter, which I know is also to be written (by me at this point... :))

Gabs

DCP User Experience (UX) Activities and Insights

User Experience (UX) Research, Design and Evaluation activities SHOULD be part of every aspect of the Product Life Cycle, informing overall product strategy, product roadmap, epics and user stories.

It is the responsibility of the respective teams to ensure they engage in User Experience (UX) activities throughout the Product Life Cycle. The UX team will either lead, or provide training and consultancy on some of these activities.

Insights and recommendations from UX activities SHOULD be shared and acted upon throughout the Product Life Cycle. A summary of the insights from various UX activities should be delivered at every iteration of the Product Roadmap, so that they can inform decisions about future changes or developments.

In addition to the above regular UX activities, UX team MAY create a wider DCP user survey on an annual basis to collect additional attitudes from the scientific community.

Reply:
@gabsie - edited and added in a section preceding the Roadmap and Planning sections to stress the importance of the UX role. I also deleted the detailed section on UX Surveys. Changes - 5049f976ea52739ff1ba03c2f90250bdfae55347 - When I add the transition from annual to quarterly roadmap updates, I'll remove any other rough edges. Cara and I appreciate you providing this contribution for us.